WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ItemToFulfillRow.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+/// Represent a row of a Product Item that should be fulfilled
+///
+struct ItemToFulfillRow: View, Identifiable {
+    let id = UUID()
+    let title: String
+    let subtitle: String
+
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading,
+                   spacing: 8) {
+                Text(title).font(.body)
+                Text(subtitle)
+                    .font(.footnote)
+                    .foregroundColor(Color(.textSubtle))
+            }.padding([.leading, .trailing], 16)
+            Spacer()
+        }.padding([.top, .bottom], 10)
+        .frame(minHeight: 64)
```

Comment:
Same question about having these magic numbers in enums. Just pointing them out in case we decide to change it, we don't miss them.